lib/beaker/logger.rb

```diff
@@ -297,14 +305,18 @@ def success *args
     # @param args[Array<String>] Strings to be reported
     def notify *args
       return unless is_notify?
-      optionally_color BRIGHT_WHITE, *args
+      if @ci_build
```

Comment:
@Iristyle I wasn't trying to fundamentally adjust the way it is now. When in Rome.